### DISCUSSION TITLE
koboldcpp as ModelProvider ???

### DISCUSSION THREAD

**Discussion by UnsGentoals** [datacrystals/AIStoryWriter#57] [Ideas]
https://github.com/LostRuins/koboldcpp

**Comment by datacrystals** [comment]
Hi there, we'd love to support that backend - is there a python package that allows for it to be called via some API?

If you join our discord, happy to discuss more there!

**Reply by UnsGentoals** [reply]
https://lite.koboldai.net/koboldcpp_api

**Reply by UnsGentoals** [reply]
http://localhost:5000/koboldcpp_api
also, you can get them from your local server.

**Reply by LoggeL** [reply]
As it's open ai compatible, implementing that should support it automatically
